frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -0,0 +1,338 @@
+  return (
+    `<ul>
+      ${resources.map((resource) => (
+      `<li><a href="${resource}">Link</a></li>`
+    )).join('')}
+    </ul>`
+  );
+}
+
+export default function ApprovedActivityReport({ match, user }) {
+  const [notAuthorized, setNotAuthorized] = useState(false);
+  const [somethingWentWrong, setSomethingWentWrong] = useState(false);
+  const [displayId, setDisplayId] = useState('');
+  const [recipientType, setRecipientType] = useState('Grantee');
+  const [creator, setCreator] = useState('');
+  const [collaborators, setCollaborators] = useState([]);
+  const [approvingManagers, setApprovingManagers] = useState('');
+  const [attendees, setAttendees] = useState('');
+  const [participantCount, setParticipantCount] = useState(0);
+  const [reasons, setReasons] = useState('');
+  const [programType, setProgramType] = useState('');
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+  const [duration, setDuration] = useState(0);
+  const [recipients, setRecipients] = useState('');
+  const [method, setMethod] = useState('');
+  const [requester, setRequester] = useState('');
+  const [topics, setTopics] = useState('');
+  const [ECLKCResources, setECLKCResources] = useState('');
+  const [nonECLKCResourcesUsed, setNonECLKCResourcesUsed] = useState('');
+  const [attachments, setAttachments] = useState([]);
+  const [context, setContext] = useState('');
+  const [goalsAndObjectiveHeadings, setGoalsAndObjectiveHeadings] = useState([]);
+  const [goalsAndObjectives, setGoalsAndObjectives] = useState([]);
+  const [managerNotes, setManagerNotes] = useState('');
+  const [creatorNotes, setCreatorNotes] = useState('');
+  const [successfullyCopiedClipboard, setSuccessfullyCopiedClipboard] = useState(false);
+
+  useEffect(() => {
+    const allowedRegions = allRegionsUserHasPermissionTo(user);
+
+    getReport(match.params.activityReportId).then((report) => {
+      if (!allowedRegions.includes(report.regionId)) {
+        setNotAuthorized(true);
+        return;
+      }
+
+      // first table
+      let recipientTypeLabel = report.activityRecipients[0].grantId ? 'Grantee' : 'Non-grantee';
+      if (report.activityRecipients.length > 1) {
+        recipientTypeLabel = `${recipientTypeLabel}s`;
+      }
+      setRecipientType(recipientTypeLabel);
+      const arRecipients = report.activityRecipients.map((arRecipient) => arRecipient.name).join(', ');
+      setRecipients(arRecipients);
+      setDisplayId(report.displayId);
+      setCreator(report.author.fullName);
+      setCollaborators(report.collaborators);
+      setApprovingManagers(report.approvingManager.fullName);
+      setAttendees(report.participants.join(', '));
+      setParticipantCount(report.numberOfParticipants);
+      setReasons(report.reason.join(', '));
+      setProgramType(report.programTypes.join(', '));
+      setStartDate(report.startDate);
+      setEndDate(report.endDate);
+      setDuration(report.duration);
```

Comment:
Spec shows 'hours' after the number again not sure if something changed in spec.